crates/hdk/src/lib.rs

```diff
@@ -2,16 +2,20 @@
 //!
 //! Functions of a Holochain application (hApp) can be organized into reusable components. In Holochain terminology these components are called "zomes".
 //! One or multiple zomes are compiled into a WebAssembly (WASM) binary, referred to as a DNA. All of the DNAs of an application are bundled to a hApp.
+//! In short, that structure is __hApp -> DNA -> zome -> function__.
+//!
+//! hApps are required to produce and validate data deterministically. There's a data model and a domain logic part to each hApp. In Holochain, the
```

Comment:
I'm happy to see you using this kind of language. Does it feel like it works consistently for you? To be honest I'm just testing it out and I'm not sure yet whether it fits or whether there are too many assumptions that devs will carry over from MVC.

Reply:
I read further and you do a great job of defining what 'domain logic' is in terms of a coordinator zome. Maybe a similar treatment somewhere (here?) for integrity zomes? Something like

> Integrity zomes describe a hApp's domain model by defining a set of entry and link types and providing a validation callback function that checks the integrity of any operations that manipulate data of those types.